### PR TITLE
UI order setting screens

### PR DESCRIPTION
**Pull request by sepehrheravi**

### CODE REVIEW

File: components/BrandHeader.js [hunk 1]
```diff
@@ -14,28 +14,26 @@ export default BrandHeader;
 
 const styles = StyleSheet.create({
   header: {
+    display: 'felx',
```
Comment (gsarthakdev) [inline components/BrandHeader.js:17]: Typo

File: constants/styles.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 export const GlobalStyles = {
   colors: {
+    deviderColor: "#EEEEEE",
```
Comment (gsarthakdev) [inline constants/styles.js:3]: Typo

**Review by gsarthakdev — CHANGES_REQUESTED** [review]
Please see attached comments.

**Review by sepehrheravi — COMMENTED** [review]
All the requested changes were made

**Review by gsarthakdev — APPROVED** [review]

**Pull request merged**